common/shlibs

```diff
@@ -2924,6 +2924,17 @@ libkj-http-0.9.1.so capnproto-0.9.1_1
 libkj-async-0.9.1.so capnproto-0.9.1_1
 libkj-test-0.9.1.so capnproto-0.9.1_1
 libkj-0.9.1.so capnproto-0.9.1_1
+libpng12.so.0 libpng12-1.2.56_2
+libpng.so.3 libpng12-1.2.56_2
+libcapnpc-0.10.2.so capnproto-0.10.2_1
+libcapnp-json-0.10.2.so capnproto-0.10.2_1
+libcapnp-rpc-0.10.2.so capnproto-0.10.2_1
+libcapnp-websocket-0.10.2.so capnproto-0.10.2_1
+libcapnp-0.10.2.so capnproto-0.10.2_1
+libkj-http-0.10.2.so capnproto-0.10.2_1
+libkj-async-0.10.2.so capnproto-0.10.2_1
+libkj-test-0.10.2.so capnproto-0.10.2_1
+libkj-0.10.2.so capnproto-0.10.2_1
```

Comment:
libpng12 is not touched in this template, is this a mistake? Also, 0.9.1.so won't be available anymore, so these lines should be replaced, not added.

Reply:
Hum sorry, it's a git rebase mistake.

Reply:
Fixed!